olp-cpp-sdk-dataservice-read/src/generated/api/MetadataApi.cpp

```diff
@@ -55,110 +55,99 @@ namespace dataservice {
 namespace read {
 using namespace olp::client;
 
-CancellationToken MetadataApi::GetLayerVersions(
+MetadataApi::LayerVersionsResponse MetadataApi::GetLayerVersions(
     const OlpClient& client, int64_t version,
     boost::optional<std::string> billingTag,
-    const LayerVersionsCallback& layerVersionsCallback) {
+    client::CancellationContext context) {
   std::multimap<std::string, std::string> headerParams;
-  headerParams.insert(std::make_pair("Accept", "application/json"));
+  headerParams.emplace("Accept", "application/json");
```

Comment:
Update code to google code style rules

```suggestion
  header_params.emplace("Accept", "application/json");
```